src/cdk/table/cell.ts

```diff
@@ -59,6 +66,26 @@ export class CdkColumnDef {
   }
   _name: string;
 
+  /** Whether this column should be sticky positioned on the start of the row */
+  @Input('sticky')
```

Comment:
The argument to `@Input()` is unnecessary when it matches the property name (here and below)

cc @crisbeto might be another fun lint check :)
(also, at some point we should probably think about upstreaming these to codelyzer)